doc/examples/applications/plot_solidification_tracking.py

```diff
@@ -0,0 +1,278 @@
+"""
+========================================
+Track solidification of a metallic alloy
+========================================
+
+In this example, we identify and track the solid-liquid (S-L) interface in a
+nickel-based alloy undergoing solidification. Tracking the solidification over
+time enables the calculatation of the solidification velocity. This is
+important to characterize the solidified structure of the sample and will be
+used to inform research into additive manufacturing of metals. The image
+sequence was obtained by the Center for Advanced Non-Ferrous Structural Alloys
+(CANFSA) using synchrotron x-radiography at the Advanced Photon Source (APS)
+at Argonne National Laboratory (ANL). This analysis was first presented at
+a conference [1]_.
+
+.. [1] Corvellec M. and Becker C. G. (2021, May 17-18)
+       "Quantifying solidification of metallic alloys with scikit-image"
+       [Conference presentation]. BIDS ImageXD 2021 (Image Analysis Across
+       Domains). Virtual participation.
+       https://www.youtube.com/watch?v=cB1HTgmWTd8
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.polynomial import polynomial
```

Comment:
This import is used only once... How about calling `np.polynomial.polyfit()`?

Reply:
It would actually be `np.polynomial.polynomial.polyfit()` so the length is the reason for the alias, but it is only used once so maybe you're right that it's better to use the full module name